test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json

```diff
@@ -1061,11 +1059,11 @@
           }
         }
       },
-      "Microsoft.Bcl.AsyncInterfaces/6.0.0": {
+      "Microsoft.Bcl.AsyncInterfaces/8.0.0": {
         "runtime": {
           "lib/netstandard2.1/Microsoft.Bcl.AsyncInterfaces.dll": {
-            "assemblyVersion": "6.0.0.0",
-            "fileVersion": "6.0.21.52210"
+            "assemblyVersion": "8.0.0.0",
+            "fileVersion": "8.0.23.53103"
```

Comment:
Please note that this is a major version change brought in by the new version of Azure.Core.

Reply:
Per offline discussion, it was noted that this is a private assembly, so this does not pose any specific risk to other components. .NET also treats this as highly backwards compatible.